### DISCUSSION TITLE
Sample Test Data

### DISCUSSION THREAD

**Discussion by amr-cloudforce** [provinzio/CoinTaxman#168] [General]
Hi, 
 The export of bitpanda in 2024 unfortunately does not work. Is there any sample data that will work for sure for the sake of comparison/debugging?

**Comment by provinzio** [comment]
I do not use Bitpanda myself and therefore cannot send you any Bitpanda related sample data. 

Perhaps you could ask a Bitpanda user or look into
#165.
